source/_integrations/uptime.markdown

```diff
@@ -10,7 +10,7 @@ ha_quality_scale: internal
 ha_domain: uptime
 ---
 
-The `uptime` sensor platform displays the time since the last Home Assistant restart.
+The `uptime` sensor platform displays the timestamp (date and time) when Home Assistant was last started.
```

Comment:
The "display" part might be a bit ambiguous as well, as it might be displayed in a UI relatively as well. So, a small suggestion to avoid that:

```suggestion
The `uptime` sensor stores the timestamp (date and time) when Home Assistant was last started.
```